examples/MamaDuckVerbose/MamaDuckVerbose.ino

```diff
@@ -37,33 +44,54 @@ void loop() {
   
   //Handle OTA requests
   ArduinoOTA.handle();
+  
+  //Get flag if LoRa packet received
+  if(duck.getFlag()) {
```

Comment:
Is this call blocking? 
If so we should indicate it in the comment. Something like:
`// this blocks until a LoRa packet is received`